tune_sklearn/tune_search.py

```diff
@@ -80,15 +80,10 @@ class TuneSearchCV(TuneBaseSearchCV):
         n_iter (int): Number of parameter settings that are sampled.
             n_iter trades off runtime vs quality of the solution.
             Defaults to 10.
-        scoring (str, `callable`, `list`, `tuple`, `dict`
-            or None): A single string (see Scikit-Learn documentation
-            on `scoring_parameter`) or a callable
+        scoring (str, `callable`, or None): A single string (see 
+            Scikit-Learn documentation on `scoring_parameter`) or a callable
             to evaluate the predictions on the test set.
-            For evaluating multiple metrics, either give a list of (unique)
-            strings or a dict with names as keys and callables as values.
-            NOTE that when using custom scorers, each scorer should return a
-            single value. Metric functions returning a list/array of values
-            can be wrapped into multiple scorers that return one value each.
```

Comment:
I removed, "Metric functions returning a list/array of values can be wrapped into multiple scorers that return one value each." but I'm not entirely sure if this actually hasn't been implemented in tune-sklearn.

Reply:
Yes, we haven't implemented this yet. Thanks!